CryptoPrices/Sources/MyCoin/Sources/MyCoin/TimeFrameSectionView/TimeFrameItem.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  TimeFrameItem.swift
+//  MyCoin
+//
+//  Created by Khanh on 04/01/2023.
+//
+
+import DomainTestHelpers
+import Entities
+import Foundation
+
+public struct TimeFrameItem: Identifiable, Equatable {
+
+    public let id = UUID()
+    public let timeFrame: TimeFilter
+    public var title: String {
+        switch timeFrame {
+        case .oneDay: return Strings.MyCoin.TimeFrame.oneDayText
+        case .oneWeek: return Strings.MyCoin.TimeFrame.oneWeekText
+        case .oneMonth: return Strings.MyCoin.TimeFrame.oneMonthText
+        case .oneYear: return Strings.MyCoin.TimeFrame.oneYearText
+        case .fiveYears: return Strings.MyCoin.TimeFrame.fiveYearText
+        }
+    }
+
+    public static func == (lhs: TimeFrameItem, rhs: TimeFrameItem) -> Bool {
+        lhs.timeFrame == rhs.timeFrame
+    }
+}
```

Comment:
The `TimeFrameItem` seems to be used internally only, I think it's fine to omit `public` marks 🙏